### PR TITLE
Feature/log tags

### PR DESCRIPTION
**Pull request by rick-foxcroft**
Tag names are now included in log files.

### CODE REVIEW

File: src/domain/reaper.py [hunk 1]
```diff
@@ -36,7 +37,7 @@ def is_instance_idle(self, instance):
         metrics = self.cloudwatch.get_average_metrics(instance['InstanceId'])
         cpu_util = metrics['AvgCPUUtilisation']
         net_out = metrics['AvgNetworkOut']
-        tags = instance['Tags']
+        tags = dumps(instance['Tags'])
```
Comment (temyers) [inline src/domain/reaper.py]: Where is the test?

Comment (rick-foxcroft) [reply src/domain/reaper.py]: Test has been added.

**Comment by temyers** [comment]
Can you rebase + squash your commits so that the merge commit is a single commit, to keep the history clean.

e.g. `rebase -i HEAD~4`

**Comment by rick-foxcroft** [comment]
@temyers I have no conflicts locally after `git pull origin <branch>`

**Pull request merged**